From 7d271e29321809af153b4e7b181785421828bcb8 Mon Sep 17 00:00:00 2001
From: Tim Schumacher <timschumi@gmx.de>
Date: Sun, 27 Jan 2019 09:42:11 +0100
Subject: [PATCH] Bump Security Patch Level to 2018-12-01

CVE-2018-9548	A-112555574	High
CVE-2018-9549	A-112160868	Critical Change 236189
CVE-2018-9552	A-113260892	Critical
CVE-2018-9553	A-116615297	High
CVE-2018-9554	A-114770654	High
CVE-2018-9555	A-112321180	Critical
CVE-2018-9557	A-35385357*	High
CVE-2018-9558	A-112161557	High	 Change 236193
CVE-2018-9559	A-112731440	High	 Change 236195
CVE-2018-9566	A-74249842	High	 Change 236206

CVE-2018-9560	A-79946737	High	 Change 236196
CVE-2018-9562	A-113164621	High	 Change 236197

Reference version: 7.0

Change-Id: I3cf28041b2f5eadf52bf10bc658fe390daf35a15
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 1701a75..9ab8ab9 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -81,7 +81,7 @@ ifeq "" "$(PLATFORM_SECURITY_PATCH)"
   # Can be an arbitrary string, but must be a single word.
   #
   # If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-  PLATFORM_SECURITY_PATCH := 2018-11-01
+  PLATFORM_SECURITY_PATCH := 2018-12-01
 endif
 
 ifeq "" "$(BUILD_ID)"
-- 
2.7.4

